codespell_lib/data/dictionary.txt

```diff
@@ -11235,7 +11235,7 @@ exeptions->exceptions
 exeqution->execution
 exerbate->exacerbate
 exerbated->exacerbated
-exerciese->exercises
+exerciese->exercise
 exercize->exercise
```

Comment:
Would you mind doing the same for this one as well?

Reply:
done in second commit